entity-framework/core/modeling/index.md

```diff
@@ -33,13 +33,53 @@ Then just invoke the `Configure` method from `OnModelCreating`.
 
 [!code-csharp[Main](../../../samples/core/Modeling/Misc/EntityTypeConfiguration.cs?Name=ApplyIEntityTypeConfiguration)]
 
+#### Applying all configurations in an assembly
+
 It is possible to apply all configuration specified in types implementing `IEntityTypeConfiguration` in a given assembly.
 
 [!code-csharp[Main](../../../samples/core/Modeling/Misc/EntityTypeConfiguration.cs?Name=ApplyConfigurationsFromAssembly)]
 
 > [!NOTE]
 > The order in which the configurations will be applied is undefined, therefore this method should only be used when the order doesn't matter.
 
+#### Using `EntityTypeConfigurationAttribute` on entity types
+
+Rather than explicitly calling `Configure`, an <xref:Microsoft.EntityFrameworkCore.EntityTypeConfigurationAttribute > can instead be placed on the entity type such that EF Core can find and use appropriate configuration. For example:
```

Comment:
```suggestion
Rather than explicitly calling `Configure`, an <xref:Microsoft.EntityFrameworkCore.EntityTypeConfigurationAttribute> can instead be placed on the entity type such that EF Core can find and use appropriate configuration. For example:
```